microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -116,6 +116,37 @@ Use this feature to measure distinct traffic from your Store-installed PWA.  Bas
 This feature was first introduced in Microsoft Edge version 91, and the DOM API was introduced in Microsoft Edge version 93.
 
 
+<!-- ====================================================================== -->
+## Use locale-specific domains without displaying them
+
+By default, store-installed PWAs display an unexpected additional UI showing the URL and page title when the app is redirected to a locale-specific domain.  This happens because the navigation to the locale-specific domain is considered "out-of-scope".  However, you can prevent this UI from being displayed, by specifying locale-specific origins that are associated with the PWA.
+
+### Domain redirection with browser-installed PWAs
+
+A Web App Manifest is tied to a single domain.  However, some PWAs use locale-specific domains for their customers in specific regions of the globe.  When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (for example, contoso.com) to a locale-specific domain (for example, contoso.co.ke), because the redirect happens during initial load of that website.
+
+Customers who install the PWA from Microsoft Edge would therefore install the PWA from the locale-specific domain.  Subsequent launches of the PWA go directly to that locale-specific domain, instead of first going to the principal domain.
+
+### Domain redirection with store-installed PWAs
+
+Store-installed PWAs, however, have a hard-coded start URL that is pointed at the principal domain.  When the PWA is launched, the PWA initially navigates to the principal domain, and then a customer may (as necessary) be redirected to their locale-specific domain. If that redirection occurs, the navigation is considered "out of scope" and the app UI will display the URL and title at the top of the web page.
```

Comment:
"at the top of the web page" - in the titlebar?  can we clarify the description or add a screenshot?